### PR TITLE
Add unit test for NoneProbeArrayStrategy

### PR DESCRIPTION
**Pull request by Godin**

### CODE REVIEW

File: org.jacoco.core.test/src/org/jacoco/core/internal/instr/NoneProbeArrayStrategyTest.java [hunk 1]
```diff
@@ -0,0 +1,49 @@
...
+		try {
+			strategy.storeInstance(null, false, 0);
+			fail("exception expected");
+		} catch (final UnsupportedOperationException e) {
+			// expected
+		}
+	}
+
+	@Test
+	public void addMembers_should_not_add_memebers() {
```
Comment (marchof) [inline org.jacoco.core.test/src/org/jacoco/core/internal/instr/NoneProbeArrayStrategyTest.java]: Typo: memebers --> members

File: org.jacoco.core.test/src/org/jacoco/core/internal/instr/NoneProbeArrayStrategyTest.java [hunk 2]
```diff
@@ -0,0 +1,49 @@
...
+public class NoneProbeArrayStrategyTest {
+
+	private NoneProbeArrayStrategy strategy;
+
+	@Before
+	public void setup() {
+		strategy = new NoneProbeArrayStrategy();
+	}
+
+	@Test
```
Comment (marchof) [inline org.jacoco.core.test/src/org/jacoco/core/internal/instr/NoneProbeArrayStrategyTest.java]: Why not `@Test(expected=UnsupportedOperationException.class)`?

Comment (Godin) [reply org.jacoco.core.test/src/org/jacoco/core/internal/instr/NoneProbeArrayStrategyTest.java]: @marchof for consistency with many other tests

File: org.jacoco.core.test/src/org/jacoco/core/internal/instr/NoneProbeArrayStrategyTest.java (hunk 1 above)
Comment (Godin) [reply org.jacoco.core.test/src/org/jacoco/core/internal/instr/NoneProbeArrayStrategyTest.java]: fixed

File: org.jacoco.core.test/src/org/jacoco/core/internal/instr/NoneProbeArrayStrategyTest.java (hunk 2 above)
Comment (marchof) [reply org.jacoco.core.test/src/org/jacoco/core/internal/instr/NoneProbeArrayStrategyTest.java]: Hm, if we don't need additional verification we typically use annotation, e.g. `ClassInstrumenterTest`.

Comment (Godin) [reply org.jacoco.core.test/src/org/jacoco/core/internal/instr/NoneProbeArrayStrategyTest.java]: changed

File: org.jacoco.core.test/src/org/jacoco/core/internal/instr/NoneProbeArrayStrategyTest.java [hunk 3]
```diff
@@ -0,0 +1,43 @@
...
+ *******************************************************************************/
+package org.jacoco.core.internal.instr;
+
+import static org.junit.Assert.assertEquals;
+
+import org.junit.Before;
+import org.junit.Test;
+import org.objectweb.asm.tree.ClassNode;
+
+public class NoneProbeArrayStrategyTest {
```
Comment (marchof) [inline org.jacoco.core.test/src/org/jacoco/core/internal/instr/NoneProbeArrayStrategyTest.java]: Out test typically name the test target:

    /**
     * Unit tests for {@link NoneProbeArrayStrategy}.
     */

Comment (Godin) [reply org.jacoco.core.test/src/org/jacoco/core/internal/instr/NoneProbeArrayStrategyTest.java]: added

**Pull request merged**